assets/css/admin.scss

```diff
@@ -2651,6 +2651,23 @@ body.everest-forms-builder-setup,
 			border: 1px solid #ddd;
 			font-size: 14px;
 		}
+
+		.recaptcha-version-options {
```

Comment:
This should be `.recaptcha-type-option`, my friend :)

Reply:
Fixed :)